components/MenuManagerBundle/src/bundle/Resources/encore/ez.config.js

```diff
@@ -12,6 +12,14 @@ const path = require('path')
 
 module.exports = (Encore) => {
   Encore.addEntry('ezplatform-admin-ui-modules-menu-manager-js', [
-    path.resolve(__dirname, '../../../modules/menu-manager/menu.manager.renderer.js')
+    path.resolve(__dirname, '../../../modules/menu-manager/menu.manager.renderer.js'),
+    path.resolve(__dirname, '../../../../../../ezplatform/vendor/ezsystems/ezplatform-admin-ui/src/bundle/Resources/public/js/scripts/button.state.toggle.js')
```

Comment:
Im having a problem with relative path to existing file in other bundles, because our bundles in this repository are outside the vendor dir.
I tried creating symlink in the vendor dir and changing the psr-4 path but it doesn't work either as the path used is the symlink resolved path.
The only way i think would solve the problem would be using a docker container and mount the component dir into the vendor dir.
Any thought @Plopix @yhanini ?

Reply:
same problem here : https://github.com/Novactive/Nova-eZPlatform-Bundles/pull/12/files#diff-c9f194e1c191dbb4ec9a91b663f1ad46R14